resources/lang/en/multiplayer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+return [
+    'empty' => 'No osu!lazer multiplayer games played yet!',
```

Comment:
```suggestion
    'empty' => 'No osu!(lazer) multiplayer games played yet!',
```

Trying not to make the "osu!lazer" term stick since it's only a codename and will eventually get dropped.

I guess if we end up having this in more areas, it may be better to add it as a blurb text (or tooltip) instead? Something like "This data is currently only populated from newer releases of osu!".